jekyll/_cci2/server-3-install-build-services.adoc

```diff
@@ -167,6 +167,66 @@ Provided in the output from `terraform apply`
 * *Nomad Server Certificate Authority (CA) Certificate (required)* - 
 Provided in the output from `terraform apply`
 
+==== Nomad Autoscaler (Optional)
+Nomad provides a utility called to automatically scale up or down your nomad clients provided your clients are managed a cloud providers autoscaling resource. With Nomad Autoscaler, you only need to provide permission for the utility to manage your autoscaling resource and where it is located. You can enable this resource via KOTS, which will deploy the nomad autoscaler service along with your nomad servers. Below we will go through how to setup nomad autoscaler for your provider.
+
+If you do not require this service then feel free to jump tp the *Save config* button to update your installation and re-deploy server.
+
+===== AWS
+. Create an IAM user/role and policy for Nomad Autoscaler. You may take one of the following approaches:
+  * Our link:https://github.com/CircleCI-Public/server-terraform/tree/main/nomad-aws[nomad module] creates an IAM user and outputs the keys if you set variable `nomad_auto_scaler = true`. You may reference the example in the link for more details. If you've already created the clients, you can simply update the variable and run `terraform apply`. The created user's access key and secret will be available in terraform's output.
+  * You may also create a nomad IAM user manually. The user will need an IAM policy with the following rules attached:
+[source, json]
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "autoscaling:CreateOrUpdateTags",
+                "autoscaling:UpdateAutoScalingGroup",
+                "autoscaling:TerminateInstanceInAutoScalingGroup"
+            ],
+            "Resource": "<<Your Autoscaling Group ARN>>"
+        },
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": [
+                "autoscaling:DescribeScalingActivities",
+                "autoscaling:DescribeAutoScalingGroups"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+. Set Nomad Auto-Scaler to `enabled`
+. Set Max Node Count* - This will overwrite what is currently set as the max for you ASG. It is recommended to keep this value and what was set in your terraform as the same.
```

Comment:
```suggestion
. Set Max Node Count* - This will overwrite what is currently set as the max for you ASG. It is recommended to keep this value and what was set in your Terraform as the same.
```